packages/jsapi-utils/src/TableUtils.test.ts

```diff
@@ -11,6 +11,7 @@ import {
 } from '@deephaven/filters';
 import TableUtils from './TableUtils';
 import DateUtils from './DateUtils';
+// eslint-disable-next-line import/no-relative-packages
 import IrisGridTestUtils from '../../iris-grid/src/IrisGridTestUtils';
```

Comment:
IrisGridTestUtils could likely be relocated. Doesn't need to be this change though.